debian/control

```diff
@@ -274,7 +274,7 @@ Build-Profiles: <!stage1>
 Architecture: amd64 armhf arm64 ppc64el s390x
 Section: kernel
 Priority: optional
-Provides: linux-image, fuse-module, kvm-api-4, redhat-cluster-modules, ivtv-modules, virtualbox-guest-modules [amd64], ${linux:rprovides}
+Provides: linux-image, fuse-module, kvm-api-4, redhat-cluster-modules, ivtv-modules, virtualbox-guest-modules [amd64], wireguard-modules (=1.0.0) ${linux:rprovides}
```

Comment:
There used to be a comma between every item in this list, including `${linux:rprovides}`. Now there's no comma before `${linux:rprovides}`. Is that okay? I see that the build worked on Jammy, but the Debian docs do [specify](https://www.debian.org/doc/debian-policy/ch-relationships.html#syntax-of-relationship-fields) that packages in this field should be comma-separated.

Reply:
Added it with [1244faa](https://github.com/pop-os/linux/pull/243/commits/1244faa49c4c783a7fe521303f4fc4b6b1e4fc7c). We'll test this once it's built again.